tests/ReportTest.php

```diff
@@ -298,4 +298,32 @@ public function testGetSummaryDuplicate()
 
         $this->assertSame(['message' => 'bar', 'severity' => 'warning'], $this->report->getSummary());
     }
+
+    /**
+     * Testing handled/unhandled.
+     */
+    public function testDefaultsToHandled()
```

Comment:
Can we add defaultseverity tests? Do we have a ticket to add code coverage stats to the tests?

Reply:
I've added some default Severity tests in